closure/goog/dom/forms_test.js

```diff
@@ -417,6 +417,26 @@ function testSetValueInput() {
   assertEquals('', goog.dom.forms.getValue(el));
 }
 
+function testGetValueMeter() {
```

Comment:
Does this work correctly in IE8?  My understanding is that IE8 doesn't natively support meter or progress elements, so I was expecting this to bail out early, say if el was undefined, or if its tagname != 'meter'.  Is that not an issue here?